controls/radrichtextbox/document-elements/features-section.md

```diff
@@ -82,7 +71,7 @@ Here is a list of these properties:
 * __PageMargin__: Represents the margin towards the edges of the page when in __Paged__ mode.
             
 
-#### __C#__
+#### __[C#] Example 3: Change the Page Margin of a Section__

```

Comment:
Add the snippets for examples 3,4 and 5 in the bullets by starting each row with a tab.